### ISSUE TITLE
Missing r.js breaks build TTW

### ISSUE THREAD

**Issue by davilima6** [plone/plone.staticresources#58]
It seems the upgrade step `Update the ``last_compilation`` date of all bundles` added in `plone.staticresources@1.2.0` was not enough to bring in the missing `r.js` script. Possibly because the `r.js` folder is missing from [src/plone/staticresources/static/components/](https://github.com/plone/plone.staticresources/tree/master/src/plone/staticresources/static/components)?

## Steps to reproduce
1. Create a new Plone instance of `buildout.coredev`
1. In Resources Registry, enable `Development Mode` and `Save`
1. Open Developer Console, click on `Build` button for any bundle and after confirming the modal see:

```
GET http://localhost:8080/Plone2/++plone++static/components/r.js/dist/r.js net::ERR_ABORTED 404 (Not Found)
Uncaught TypeError: Cannot read property 'optimize' of undefined
    at Object.onLoad (resourceregistry-compiled.js:33643)
    at window.IFrame.load (resourceregistry-compiled.js:33406)
```

**Comment by thet** [comment]
Ha! Got it. The original fix contained these Repos:

https://github.com/plone/Products.CMFPlone/pull/2970
https://github.com/plone/plone.staticresources/pull/45
https://github.com/plone/plone.app.upgrade/pull/218

The CMFPlone part was merged into master, which became Plone 6.0.

So the fix was actually missing for Plone 5.2.

This fixes it: https://github.com/plone/Products.CMFPlone/pull/3011

Thanks for reporting and cross-checking!

**Comment by thet** [comment]
Still broken. See: https://github.com/plone/Products.CMFPlone/pull/3011#issuecomment-570786863
Fix underway.
